tests/integration/workspace_access/test_redash.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+from databricks.sdk.service import sql
+from databricks.sdk.service.sql import ObjectTypePlural
+from integration.conftest import list_equal_unordered
+
+from databricks.labs.ucx.workspace_access import redash
+from databricks.labs.ucx.workspace_access.groups import (
+    GroupMigrationState,
+    MigrationGroupInfo,
+)
+
+
+@pytest.mark.skip("Make sure the environment doens't have any query created otherwise it will fail")
+def test_one_asset_should_have_permission_recplicated_to_backup_group(ws, make_query, make_ucx_group, make_group):
+    ws_group, acc_group = make_ucx_group()
+    backup_group_name = ws_group.display_name + "-backup"
+    backup_group = make_group(display_name=backup_group_name)
+    query = make_query()
+
+    migration_state = GroupMigrationState()
+    migration_state.add(group=MigrationGroupInfo(workspace=ws_group, account=acc_group, backup=backup_group))
+
+    ws.dbsql_permissions.set(
+        object_type=ObjectTypePlural.QUERIES,
+        object_id=query.id,
+        access_control_list=[
+            sql.AccessControl(group_name=ws_group.display_name, permission_level=sql.PermissionLevel.CAN_RUN)
+        ],
+    )
+
+    redash_acl_listing = [redash.redash_listing_wrapper(ws.queries.list, sql.ObjectTypePlural.QUERIES)]
+    sql_support = redash.SqlPermissionsSupport(ws, redash_acl_listing)
+
+    tasks = list(sql_support.get_crawler_tasks())
+    # Only one query exist in the workspace, so one crawler task
+    assert len(tasks) == 1
```

Comment:
This will fail when multiple integration suits are run in parallel or there's some residue from previous tests